resource/doc/libvmaf.md

```diff
@@ -54,9 +54,58 @@ Using FFmpeg with libvmaf is very powerful, as you can create complex filters to
+Here `main.mpg` is a downscaled and encoded video and `ref.mpg` is its reference source at 1080p. The command scales the first input video (`0:v`) and forwards it to VMAF (`libvmaf`) with the label `main`, where it is compared against the label `ref` which corresponds to the second input reference video, `1:v`. We enforce the scaling algorithm to bicubic, since the default scaling algorithm is bilinear (also see the [techblog](https://medium.com/netflix-techblog/vmaf-the-journey-continues-44b51ee9ed12) for the recommendation on upsampling methods). Note that we need to specify the `setpts=PTS-STARTPTS` option to ensure that the PTS is correctly
+aligned. We can also the `-an` option to disregard the presence of audio.
 
-See the [FFmpeg's guide to libvmaf](https://ffmpeg.org/ffmpeg-filters.html#libvmaf), the [FFmpeg Filtering Guide](https://trac.ffmpeg.org/wiki/FilteringGuide) for more examples of complex filters, and the [Scaling Guide](https://trac.ffmpeg.org/wiki/Scaling) for information about scaling and using different scaling algorithms.
+See the [FFmpeg's guide to libvmaf](https://ffmpeg.org/ffmpeg-filters.html#libvmaf), the [FFmpeg Filtering Guide](https://trac.ffmpeg.org/wiki/FilteringGuide) for more examples of complex filters, and the [Scaling Guide](https://trac.ffmpeg.org/wiki/Scaling) for information about scaling and using different scaling algorithms. Please refer to the next section for some additional examples on more complex cases.
+
+### Use `libvmaf.a` with FFmpeg - More examples
+
+The above command line works well for most cases, but additional steps might be necessary.
+
+For example, when the frame rates between reference and encoded video do not match, e.g., when the source is 60 fps and the encode is 30 fps:
+
+```
+ffmpeg -r 1 -i main.mpg -r 2 -i ref.mpg -filter_complex \
+"[0:v]scale=1920x1080:flags=bicubic,setpts=PTS-STARTPTS[main];[1:v]select='not(mod(n,2))',setpts=PTS-STARTPTS[ref];[main][ref]libvmaf" -an -f null -
+```
+
+Here we explicitly state that the reference video has double the frame rate and also use the select filter to select half of the reference frames to calculate VMAF.
+
+If we want a specific model to be used (e.g. the `baz.pkl` model file under the `/foo/bar/` directory):
+
+```
+ffmpeg -i main.mpg -i ref.mpg -filter_complex \
+"[0:v]scale=1920x1080:flags=bicubic,setpts=PTS-STARTPTS[main];[1:v]setpts=PTS-STARTPTS[ref];[main][ref]libvmaf='model_path=/foo/bar/baz.pkl'" -an -f null -
+```
+
+If we want to specify a model that relies on bootstrapping (e.g. the 0.6.3 model file under the `/foo/bar/` directory), the `enable_confidence_interval` option needs to be set:
```

Comment:
I think it would be good to mention the default path where models are installed (`/usr/local/share/model`) in addition to just `/foo/bar`. A novice user will not know where these models are, or whether they need to be downloaded separately etc.

Reply:
Noted, will fix that.